examples/QIR/Simulation/TraceSimulator/README.md

```diff
@@ -0,0 +1,229 @@
+# The Trace Simulator
+
+A trace simulator is a quick way to provide a backend to the QIR Runtime system.
+It's function is to print out all quantum instructions it receives.
+While this can useful for debugging, it is also a simple way to connect QIR to hardware backends that do not support control flow.
```

Comment:
In that application, it's worth noting that the output from trace simulation isn't a representation of the entire quantum program, but a trajectory through the program.

Reply:
This is true, which is why I specify this applies to "straight-line" programs with no control flow.